backend/baseclass.pm

```diff
@@ -274,13 +277,9 @@ sub unlink_crash_file {
 }
 
 sub write_crash_file {
-    if (open(my $fh, ">", $iscrashedfile)) {
-        print $fh "crashed\n";
-        close $fh;
-    }
-    else {
-        warn "cannot write '$iscrashedfile'";
-    }
+    (open(my $fh, ">", $iscrashedfile));
```

Comment:
It's not magic but stupid leftover of deleting the surrounding 'if'. I will correct this. anything else?